style.css

```diff
@@ -0,0 +1,41 @@
+.calculator{
+    width: 25%;
+    height: 40%;
+    margin: 0 auto;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-auto-rows: minmax(30px, 80px);
+    grid-gap: 5px;
+    font-size: 1.5em;
+    
+}
+
+.calculator .operator{
```

Comment:
Let's also clean up by adding a space between the selectors and the curly braces too `.calculator .operator {` for the whole file please.